code/src/components/SendThought.css

```diff
@@ -0,0 +1,52 @@
+
+.send-thought-div {
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+textarea {
+  display: flex;
+    font-family: Cutive Mono;
+    font-size: 15px;
+    width: 400px;
+    height: 100px;
+    background-color: rgb(255, 240, 242);
+    padding: 10px;
+    resize: none;
+    border: 1px solid black;
+    background-color: rgb(255, 246, 246);
+    box-shadow: 5px 5px 0px 0px black;
+    margin-bottom: 5px;
+  }
+
+  textarea:focus {
+    outline: none;
+  }
+
+  .character-div {
+    display: flex;
+    justify-content: end;
+    margin-bottom: -10px;
+  }
+
+  .character-count {
+   font-size: 12px;
+   line-height: 5px;
+  }
+
+  .send-thought-btn {
+    display: flex;
+    font-family: Cutive Mono;
+    background: rgb(255, 122, 122);
+    border: 0px;
+    border-radius: 10px;
+    color: black;
+    font-size: 15px;
+    padding: 5px 15px 5px 15px;
```

Comment:
You could just write padding: 5px 15px Than would make it 5 px of padding at the top and bottom and 15px of padding to the left and right.